java/client/src/test/java/glide/connectors/resources/ThreadPoolResourceAllocatorTest.java

```diff
@@ -30,17 +28,19 @@ public void init() {
     public void getOrCreate_returns_default_after_repeated_calls() {
         ThreadPoolResource mockedThreadPoolResource = mock(ThreadPoolResource.class);
         EventLoopGroup mockedEventLoopGroup = mock(EventLoop.class);
+        @SuppressWarnings("unchecked")
```

Comment:
why?

Reply:
`Supplier` is a paramtrized type (it has `<>`), but mock returns a raw type of `Supplier` and can't return `Supplier<something>`.
Assigning `Supplier` to a `Supplier<...>` does not require a cast, but it produces a compiler warning, which we acknoledge and suppress.
If we keep using raw `Supplier` we have another warning about using raw types, which should be also suppressed.

Reply:
We can use an actual supplier instead of mocks.  We would lose the verify check, but that's not really important.